Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM golang:1.17
```

Comment:
We may want an issue solely for bumping to Go 1.17 throughout the repository.

Reply:
Is this already done? That's the version defined in the `go.mod` currently.

Reply:
Yeah whoops you're right (I should've doubled check my bad :laughing:)